module6/freethrows/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: Free Throws
+---
+
+During Intermission, you worked on earning a certification in fundamental C#.  This is one way to keep up with your technical skills during a job hunt.  Another way to stay current with your skills is by practicing 'Free Throws' - activities that will help you cement your existing skills into muscle memory.
+
+During Mod6 we will have some dedicated Free Throw time that will focus on one of three areas: programming fundamentals, OOP Practice, and SQL Practice.  
+
+<section class='note' markdown='1'>
+Keep revisiting this page - we may add more opportunities for practice as we go!
+</section>
+
+### Programming Fundamentals
+
+* [Edabit](https://edabit.com/challenges) - Work through all the c# challenges, starting with 'Very Easy' and moving steadily up to 'Expert'
```

Comment:
I might change this to "Steadily increasing in difficulty". I haven't looked at these, but I know on other platforms Expert is very hard and much more than they need.

Reply:
Totally agree that they have heard leetcode a lot - because of that I thought they might already have started to explore that, and wanted to give another option, but I think we can add leetcode to this!